Source/Core/DolphinQt2/Config/SettingsWindow.cpp

```diff
@@ -95,6 +111,8 @@ void SettingsWindow::MakeCategoryList()
   AddCategoryToList(tr("Interface"), "browse");
   AddCategoryToList(tr("Audio"), "play");
   AddCategoryToList(tr("Paths"), "browse");
+  AddCategoryToList(tr("Wii"), "Platform_Wii");
```

Comment:
You should just use `wiimote` instead of `Platform_Wii` and get rid of the extra logic above